### ISSUE TITLE
Mind a merge?

### ISSUE THREAD

**Issue by twprogrammers** [anthonycr/Lightning-Browser#135]
I just made a few mods to add a home button and a search preset for PrivateLee.com and would like to merge. I figured I would ask before I just used a merge request. I was not able to test the code due to a few missing dependencies on my end, however it should work since it did on a [fork of an older Lightning Browser](http://github.com/FunPanda08/Web-browser).

**Comment by twprogrammers** [comment]
Nearly a year later and it gets closed without a reason or response...

**Comment by anthonycr** [comment]
I apologize, I do owe you a response. I haven't been keeping up on issues as well as I should have and I forgot about this issue (and many others). I went through and decided I needed to clean up the issue tracker and start paying attention, and since this issue was very old I figured it is not relevant any more as the code difference between the branches is likely very significant and a merge would be difficult work.

To get to the content of the merge... I can't say whether I will merge something until I see the code. I do accept merge requests though, so if you have been keeping your fork up to date and want to create a pull request, I will gladly review it.
